doc-content/enterprise-only/openshift/template-deploy-externaldb-proc.adoc

```diff
@@ -44,6 +44,10 @@ complete the following additional configuration when configuring the template to
 +
 ** *KIE Server External Database User* (`KIE_SERVER_EXTERNALDB_USER`) and KIE Server External Database Password (`KIE_SERVER_EXTERNALDB_PWD`): The user name and password for the external database server
 ** *KIE Server External Database URL* (`KIE_SERVER_EXTERNALDB_URL`): The JDBC URL for the external database server
+[NOTE]
+====
+If you are using the EntrepriseDB Postgres database server, do not use an URL starting with `jdbc:edb://`. Instead, use an URL starting with `jdbc:postgresql://`. Alternatively, do not set the URL and set the host and port parameters instead.
```

Comment:
Sounds good, also, we could add another note saying that, if user wants/needs to use edb URL, he might need to provide the postgrelsql jdbc driver manually and also configure the hibernate dialect properly.

Reply:
@spolti I don't think this fits in this doc as a quick change. We can make it a separate topic in the doc or else we can make a KB article, and both of these things need to wait at least until next sprint, because of the amount of research involved in documenting this action properly. The target for this would be 7.10, most likely.

The question is what we can do as a quick change right now, pushing into previous doc versions too if needed (the same information will also be reflected in 7.9.1 release notes).

Reply:
Sounds good Misha, thanks.
feel free to apply these changes.